meteor/sofie-server/business-logic/facades/service-facade.ts

```diff
@@ -0,0 +1,32 @@
+import { RundownService } from '../services/interfaces/rundown-service'
+import { RundownEventService } from '../services/rundown-event-service'
+import { RundownTimelineService } from '../services/timeline-integration/rundown-timeline-service'
+import { RepositoryFacade } from '../../data-access/facades/repository-facade'
+import { TimelineBuilder } from '../services/interfaces/timeline-builder'
+import { TimelineBuilderImplementation } from '../services/timeline-integration/timeline-builder-implementation'
+import { RundownEventBuilderImplementation } from '../services/rundown-event-builder-implementation'
+import { RundownEventBuilder } from '../services/interfaces/rundown-event-builder'
+import { RundownLockService } from '../services/rundown-lock-service'
+
+export class ServiceFacade {
+	public static createRundownService(): RundownService {
+		// return new SofieCoreMeteorRundownService()
+		const timelineService: RundownService = new RundownTimelineService(
+			RundownEventService.getInstance(),
+			RepositoryFacade.createRundownRepository(),
+			RepositoryFacade.createTimelineRepository(),
+			RepositoryFacade.createAdLibRepository(),
+			ServiceFacade.createTimelineBuilder(),
+			ServiceFacade.createRundownEventBuilder()
+		)
+		return RundownLockService.getInstance(timelineService)
+	}
+
+	public static createTimelineBuilder(): TimelineBuilder {
+		return new TimelineBuilderImplementation()
```

Comment:
Is the `Implementation` suffix not part of the list of suffixes that are bad practice to use?

Same comment goes to the similarly returned class in the method below.

Reply:
Yes it is. The name needs to be changed. I just had a hard time finding a good name for it (which showcases that something most likely need to change here)